config/initializers/assets.rb

```diff
@@ -0,0 +1 @@
+Rails.application.config.assets.precompile += ['payments.js']
```

Comment:
Since you are using `payment.js` this should be without "s". Not `payments` but `payment`.